onnxruntime/test/providers/provider_test_utils.cc

```diff
@@ -512,12 +538,11 @@ void OpTester::Run(ExpectResult expect_result,
         EXPECT_TRUE(session_object.RegisterExecutionProvider(std::move(entry)).IsOK());
       }
 
-      ExecuteModel(*p_model, session_object, expect_result, expected_failure_string, run_options,
-                   feeds, output_names, provider_types);
+      ExecuteModel(*p_model, session_object, expect_result, expected_failure_string, run_options, feeds, output_names,
+                   provider_types);
     } else {
       for (const std::string& provider_type : all_provider_types) {
-        if (excluded_provider_types.count(provider_type) > 0)
-          continue;
+        if (excluded_provider_types.count(provider_type) > 0) continue;
 
```

Comment:
nit: harder to set a breakpoint with this formatting

Reply:
Fixed. (in all the places)